src/grammar.coffee

```diff
@@ -353,6 +353,7 @@ grammar =
 
   Import: [
     o 'IMPORT Expression',                      -> new Import $2
+    o 'IMPORT Assignable FROM Expression',      -> new Import $4, $2
```

Comment:
You need to replace `Assignable` with something new that represents the ES2015 import syntax, and `Expression` with something that only matches `'` and `"` strings (also keep in mind that interpolation cannot be used here.)